### DISCUSSION TITLE
max length of audio files in dataset  / change max duration?

### DISCUSSION THREAD

**Discussion by XanomalieX** [SWivid/F5-TTS#494] [Q&A] [answered]
Hi
I got a already transcripted and segmented dataset, but the files are 30s long
and the "Prepare Data" says: 
max sec: 24.92

is it a bad idea to change the max duration and train with this?
new transcription would take forever :-(

**Comment by SWivid** [comment, marked as answer]
https://github.com/SWivid/F5-TTS/blob/eb84cbab0f8e1b1a0dd9ca4c30fe7976ca02f60f/src/f5_tts/train/finetune_gradio.py#L803-L807
Feel free to change the threshold of duration (not sure if it's the only place needed to change)

**Reply by XanomalieX** [reply]
thank you very much for the fast response!
I was used to max 12s, so 30s does not seen like a good idea, 
but it would take months to transcribe, so i give it a try. ;-)
